app/index.html

```diff
@@ -45,18 +45,18 @@
 		</div>
 		<!-- Javascript -->
 		<script> 
-			const WebFrame = require("web-frame");
-			const RendererIPC = require("ipc");
-			const ElectronScreen = require("screen");
+			const WebFrame = require('web-frame');
+			const RendererIPC = require('ipc');
+			const ElectronScreen = require('screen');
 			const Process = require('child_process').spawn;
-			const Path = require("path");
-			const Fs = require("fs");
-			const Shell = require("shell");
-			const $ = require("jquery");
+			const Path = require('path');
+			const Fs = require('fs');
+			const Shell = require('shell');
+			const $ = require('./js/jquery.min.js');
```

Comment:
Why a minified js file instead of the node package? Same principle as BigNumber I guess. You can avoid dependency's updates breaking your usage by specifying a version number in the package.json instead of `latest` ya know.